util/test/prediff-for-slurm-srun

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+#
+# This script is a system-wide prediff for use with the slurm-srun
+# launcher.
+#
+import sys, re
+
+# These match the message blocks we want to remove.
+msgs = (
+"""srun: error: .+: task [0-9]+: Exited with exit code [0-9]+
```

Comment:
Any reason to include the newlines in here?

Reply:
The `re.sub()` is applied to the entire output file at once, so matching the whole line (including the newline) and replacing with `""` effectively removes the undesired lines.

Reply:
Gotcha